src/Connection.php

```diff
@@ -113,6 +113,11 @@ class Connection
      */
     protected static $multiByteSupport = null;
 
+    public static $validResources = array(
+        'socket' => true,
+        'stream' => true
+    );
+
```

Comment:
I was originally using the [] syntax, but the project has PHP 5.3 compatibility checks in TravisCI ;-).